pages/docs/concepts/asyncapi-document/resuable-parts.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Reusable parts
+weight: 249
+---
+
+Reusable parts in AsyncAPI provide flexibility, modularity, and code reusability. Using reusable parts makes it easier to generate code and validate the specifications. You can reuse specific document sections such as Messages or schema definitions.
+
+Reusable parts allow you to split up the AsyncAPI document into many files and reference them using the Reference Object ($ref). You can use the $ref keyword to reference the same or another local file or external URL.
```

Comment:
Sometimes you write `$ref` and sometimes only `ref`. For consistency, you must stick to one. Please update all these mentions consistently. ✌🏽